daemon/cmd/policy.go

```diff
@@ -125,6 +125,7 @@ func newPolicyTrifecta(params policyParams) (policyOut, error) {
 
 	params.Lifecycle.Append(hive.Hook{
 		OnStart: func(hc hive.HookContext) error {
+			ipc.Start()
```

Comment:
Cilium can be very sensitive in startup ordering, how does this change impact the initial trigger of IPCache state synchronization?

Separately, it seems like `newPolicyTrifecta` is used in tests, so will running a controller there have other unintended follow-on effects?

Reply:
(removed this commit)